### PR TITLE
Add Boutiques integration for `value-disables`

### PR DESCRIPTION
**Pull request by natacha-beck**
Close: #1158

### CODE REVIEW

File: BrainPortal/lib/cbrain_task_generators/templates/task_params.html.erb.erb [hunk 1]
```diff
@@ -71,6 +71,49 @@
     single_file = <%= single_file ? "\"#{single_file['id']}\"" : "nil" %>
 %%>
 
+
```
Comment (prioux) [inline BrainPortal/lib/cbrain_task_generators/templates/task_params.html.erb.erb:74]: Isn't this duplicating completely the lambdas that are already defined at line 508 (original code, line 558 in the new code) ?

File: BrainPortal/lib/cbrain_task_generators/templates/task_params.html.erb.erb [hunk 2]
```diff
@@ -136,18 +179,18 @@
   <ul class="tsk-prm-list">
     <li>
       <%%
-        input.(id, name + "[]",
+        input.((id + "_0"), name + "[]",
```
Comment (prioux) [inline BrainPortal/lib/cbrain_task_generators/templates/task_params.html.erb.erb:182]: Why the special hardcoded `_0` ?!?

Comment (prioux) [reply BrainPortal/lib/cbrain_task_generators/templates/task_params.html.erb.erb:182]: Oh I get it, the IDs are made distinct later on in the loop

File: BrainPortal/lib/cbrain_task_generators/templates/task_params.html.erb.erb [hunk 3]
```diff
@@ -71,6 +71,49 @@
...
+          if map[v].nil? then map[v] = [key] else map[v] << key end
+        end
+      end
+      map
+    end
+  end
+
+  #
+  # The actual maps: idIn -> [ids disabled/required by idIn]
+  disabledBy_map   = makeReceiverActionMap.("disables-inputs")
```
Comment (prioux) [inline BrainPortal/lib/cbrain_task_generators/templates/task_params.html.erb.erb:108]: Here you are computing maps that are never used (disabledBy_map and requires_map)

**Comment by prioux** [comment]
So can we close this PR, @natacha-beck ?

**Comment by natacha-beck** [comment]
Please let this one open for now. But no need to review.

**Comment by natacha-beck** [comment]
Duplicate of #1190

**Pull request closed without merging**